src/main/java/org/example/Basic.java

```diff
@@ -0,0 +1,8 @@
+package org.example;
+
+public class Basic implements Calculator {
```

Comment:
The name should be meaningful on its own. "BasicCalculator" would mean more than just "Basic" unless you know about how it implements Calculator under the hood.